### PR TITLE
[YUNIKORN-1511]Adding Chinese translation of Deploy to Kubernetes

### PR DESCRIPTION
**Pull request by 0lai0**
### What is this PR for?
The Chinese translation for the document Deploy to Kubernetes
https://yunikorn.apache.org/docs/developer_guide/deployment

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### What is the Jira issue?
[YUNIKORN-1511]Adding Chinese translation of Deploy to Kubernetes
*https://issues.apache.org/jira/browse/YUNIKORN-1511

### Screenshots (if appropriate)
image

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+部署YuniKorn的最简单方法是利用我们的[helm charts](https://hub.helm.sh/charts/yunikorn/yunikorn)，您可以在[这里](get_started/get_started.md)找到指南。本文档描述了手动部署YuniKorn调度器和准入控制器的过程，主要面向开发人员。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:25]: 本文档主要面向开发人员，描述了手动部署YuniKorn调度器和准入控制器的过程。

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+make image
+```
+
+此命令将构建一个镜像。该镜像将使用默认版本、镜像标签和您的构建架构进行标记。
+
+**注意** 默认构建使用硬编码的用户和标签。您 *必须* 更新 `Makefile` 中的 `IMAGE_TAG` 变量，以便推送到适当的仓库。
+
+**注意** Docker Hub 上的最新 YuniKorn 镜像不再更新，因为ASF政策的原因。因此，在部署之前，您应该在本地构建调度器镜像和Web镜像。
+
+**注意** 镜像标记包含您的构建架构。对于Intel，它将是 `amd64`，对于Mac M1，它将是 `arm64v8`。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:40]: :::note
1. 默认构建使用硬编码的用户和标签。您 *必须* 更新 `Makefile` 中的 `IMAGE_TAG` 变量，以便推送到正确的仓库。
2. Docker Hub 上使用最新标签之YuniKorn 镜像不再更新，因为 ASF 政策的原因。因此，在部署之前，您应该在本地部署调度器和 Web 镜像。
3. 镜像标记包含您的构建架构。对于Intel，它将是 `amd64`，对于Mac M1，它将是 `arm64`。
:::

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+第一步是为调度器创建RBAC角色，请参考以下步骤: [yunikorn-rbac.yaml](https://github.com/apache/yunikorn-k8shim/blob/master/deployments/scheduler/yunikorn-rbac.yaml)
+```
+kubectl create -f scheduler/yunikorn-rbac.yaml
+```
+角色是当前版本Kubernetes的要求。
+
+## 创建/更新ConfigMap
+
+在部署调度器之前，必须执行此操作。它需要一个正确设置的Kubernetes环境。这个Kubernetes环境可以是本地的或远程的。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:52]: 在部署调度器之前，必须执行此操作，正确设置Kubernetes环境。这个Kubernetes环境可以是本地的或远程的。

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+## 部署调度器
+
+可以使用以下命令部署调度器。
+```
+kubectl create -f deployments/scheduler/scheduler.yaml
+```
+
+部署将在一个Pod中运行2个容器，这些容器使用了您预先构建的Docker镜像。
+
+* yunikorn-scheduler-core (yunikorn scheduler core and shim for K8s)
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:81]: yunikorn-scheduler-core (适用于 K8s 的 yunikorn 调度器核心和shim）

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+* yunikorn-scheduler-web (web UI)
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:82]: yunikorn-scheduler-web (网页UI)

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+```yaml
+  metadata:
+    name: pod_example
+    labels:
+      applicationId: appID
+  spec:
+    schedulerName: yunikorn
+```
+
+注意：准入控制器会从用户那里抽象出`schedulerName`和`applicationId`的添加，因此会将所有流量路由到YuniKorn。如果您使用Helm chart进行部署，它将会同时安装准入控制器和调度器。否则，如果运行的工作负载中Pod规范和元数据中没有`schedulerName`和`applicationId`，则可以继续下面的步骤手动部署准入控制器。
```
Comment (0yukali0) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:99]: :::note
准入控制器会从用户那里抽象出`schedulerName`和`applicationId`的添加，因此会将所有流量路由到YuniKorn
如果您使用Helm chart进行部署，它将会同时安装准入控制器和调度器
否则，如果运行的工作负载中Pod规范和元数据中没有`schedulerName`和`applicationId`，则可以继续下面的步骤手动部署准入控制器。
:::

**Review by 0yukali0 — CHANGES_REQUESTED** [review]

**Comment by 0yukali0** [comment]
In both documents with different languages, replacing ":::note something :::" with "*note* something" marks messages.

**Review by steinsgateted — CHANGES_REQUESTED** [review]
@0lai0 @0yukali0 Thank you
Update documentation #338

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+部署YuniKorn的最简单方法是利用我们的[helm charts](https://hub.helm.sh/charts/yunikorn/yunikorn)，您可以在[这里](get_started/get_started.md)找到指南。本文档描述了手动部署YuniKorn调度器和准入控制器的过程，主要面向开发人员。
+
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:26]: There have new commit after this PR. ( 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48 )
Missing translation of 
> **Note** The primary source of deployment information is the Helm chart, which can be found at [yunikorn-release](https://github.com/apache/yunikorn-release/). Manual deployment may lead to out-of-sync configurations, see [deployments/scheduler](https://github.com/apache/yunikorn-k8shim/tree/master/deployments/scheduler)

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+**注意** Docker Hub 上的最新 YuniKorn 镜像不再更新，因为ASF政策的原因。因此，在部署之前，您应该在本地构建调度器镜像和Web镜像。
+
+**注意** 镜像标记包含您的构建架构。对于Intel，它将是 `amd64`，对于Mac M1，它将是 `arm64v8`。
+
+## 设置调度器的RBAC
+
+第一步是为调度器创建RBAC角色，请参考以下步骤: [yunikorn-rbac.yaml](https://github.com/apache/yunikorn-k8shim/blob/master/deployments/scheduler/yunikorn-rbac.yaml)
+```
+kubectl create -f scheduler/yunikorn-rbac.yaml
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:46]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48

> kubectl create -f deployments/scheduler/admission-controller-rbac.yaml

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+```
+角色是当前版本Kubernetes的要求。
+
+## 创建/更新ConfigMap
+
+在部署调度器之前，必须执行此操作。它需要一个正确设置的Kubernetes环境。这个Kubernetes环境可以是本地的或远程的。
+
+- 如果配置文件在节点上不可用，请下载它以添加到Kubernetes:
+```
+curl -o queues.yaml https://raw.githubusercontent.com/apache/yunikorn-k8shim/master/conf/queues.yaml
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:56]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48
> curl -o yunikorn-configs.yaml https://raw.githubusercontent.com/apache/yunikorn-k8shim/master/deployments/scheduler/yunikorn-configs.yaml

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+```
+- 根据需要修改 `queues.yaml` 文件的内容，然后在Kubernetes中创建ConfigMap:
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:58]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48

> - modify the content of yunikorn-configs.yaml file as needed, and create ConfigMap in kubernetes:

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+```
+kubectl create configmap yunikorn-configs --from-file=queues.yaml
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:60]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48

> kubectl create configmap yunikorn-configs --from-file=yunikorn-configs.yaml

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+```
+- 或者在Kubernetes中更新ConfigMap:
+```
+kubectl create configmap  yunikorn-configs --from-file=queues.yaml -o yaml --dry-run=client | kubectl apply -f -
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:64]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48

> kubectl create configmap yunikorn-configs --from-file=yunikorn-configs.yaml -o yaml --dry-run=client | kubectl apply -f -

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+```
+- 检查ConfigMap是否已正确创建/更新:
+```
+kubectl describe configmaps yunikorn-configs
+```
+注意：在资源分配队列规范中使用与yunikorn-defaults配置中使用的文件名相同的文件名，例如 `queues.yaml`。
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:70]: Removed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+注意：准入控制器会从用户那里抽象出`schedulerName`和`applicationId`的添加，因此会将所有流量路由到YuniKorn。如果您使用Helm chart进行部署，它将会同时安装准入控制器和调度器。否则，如果运行的工作负载中Pod规范和元数据中没有`schedulerName`和`applicationId`，则可以继续下面的步骤手动部署准入控制器。
+
+## 设置准入控制器的RBAC
+
+在部署准入控制器之前，我们必须创建其RBAC角色，请参考以下链接： [admission-controller-rbac.yaml](https://github.com/apache/yunikorn-k8shim/blob/master/deployments/scheduler/admission-controller-rbac.yaml).
+
+```
+kubectl create -f scheduler/admission-controller-rbac.yaml
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:106]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48
> kubectl create -f deployments/scheduler/admission-controller-rbac.yaml

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+## 创建Secret
+
+由于准入控制器拦截对API服务器的调用以验证/改变传入的请求，因此我们必须部署一个空的Secret，用于存储Webhook服务器的TLS证书和密钥，请参考以下链接：[admission-controller-secrets.yaml](https://github.com/apache/yunikorn-k8shim/blob/master/deployments/scheduler/admission-controller-secrets.yaml).
+
+```
+kubectl create -f scheduler/admission-controller-secrets.yaml
```
Comment (chenyulin0719) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/developer_guide/deployment.md:114]: Changed in 9a2f4a93d84b2e6c56a1c6e29a2aaaa1d66c3e48
> kubectl create -f deployments/scheduler/admission-controller-secrets.yaml

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
Some sentences are already outdated. Please update to the latest version. Thanks.

**Comment by wilfred-s** [comment]
No longer relevant after [YUNIKORN-2411](https://issues.apache.org/jira/browse/YUNIKORN-2411)

**Pull request closed without merging**